src/main/java/me/shikichee/AnchorApplication.java

```diff
@@ -7,6 +7,11 @@
 public class AnchorApplication {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.yo1000.com/coding_rules#rule_key=squid%3AS1118)